src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -68,15 +68,8 @@ public function __construct(
         $this->logger = $logger ?: new NullLogger();
     }
 
-    public function returnImage($id, $formatKey, $fileName /*, int<1, max>|null $version = null */)
+    public function returnImage($id, $formatKey, $fileName, ?int $version)
```

Comment:
This should not be nullable, but seeing that this would need a refactor of the `FormatManager` I've kept it nullable.

Reply:
no its the behaviour we want. if no specific version is requested we returning the last one. Its fine this method is null.


```suggestion
    public function returnImage($id, $formatKey, $fileName, ?int $version = null)
```